src/org/zaproxy/zap/extension/bugtracker/BugTrackerGithub.java

```diff
@@ -0,0 +1,471 @@
+	private String FIELD_USERNAME = "bugtracker.trackers.github.issue.username";
+	private String FIELD_PASSWORD = "bugtracker.trackers.github.issue.password";
+    private String FIELD_GITHUB_CONFIG = "bugtracker.trackers.github.issue.config";
+    private String titleIssue = null;
+    private String bodyIssue = null;
+    private String labelsIssue = null;
+    private JPanel configTable = null;
+    private JPanel githubPanel = null;
+    private BugTrackerGithubTableModel githubModel = null;
+    private RaiseSemiAutoIssueDialog dialog = null;
+
+    private static final Logger log = Logger.getLogger(BugTrackerGithub.class);   
+
+    public BugTrackerGithub() {
+        initializeConfigTable();
+    }
+
+    public void setDetails(Set<Alert> alerts) {
+        setTitle(alerts);
+        setBody(alerts);
+        setLabels(alerts);
+    }
+
+    public void setDialog(RaiseSemiAutoIssueDialog dialog) {
+        this.dialog = dialog;
+    }
+
+    public void initializeConfigTable() {
+        githubPanel = new BugTrackerGithubMultipleOptionsPanel(getGithubModel());
+    }
+
+    public JPanel getConfigPanel() {
+        return githubPanel;
+    }
+
+    public void createDialogs() {
+        List<BugTrackerGithubConfigParams> configs = getOptions().getConfigs();
+        Set<String> collaborators = new HashSet<String>();
+        List<String> configNames = new ArrayList<String>();
+        for(BugTrackerGithubConfigParams config: configs) {
+            configNames.add(config.getName());
+        }
+        dialog.setXWeights(0.1D, 0.9D);
+        dialog.addComboField(FIELD_GITHUB_CONFIG, configNames, "");
+        dialog.addTextField(FIELD_REPO, "");
+        dialog.addTextField(FIELD_TITLE, getTitle());
+        dialog.addMultilineField(FIELD_BODY, getBody());
+        dialog.addTextField(FIELD_LABELS, getLabels());
+        dialog.addTextField(FIELD_ASSIGNEE_MANUAL, "");
+        dialog.addComboField(FIELD_ASSIGNEE_LIST, new ArrayList<String>(), "");
+        dialog.addTextField(FIELD_USERNAME, "");
+        dialog.addTextField(FIELD_PASSWORD, "");
+        updateAssigneeList();        
+        dialog.addFieldListener(FIELD_GITHUB_CONFIG, new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                updateAssigneeList();
+            }
+        });
+    }
+
+    public void updateAssigneeList() {
+        try {
+            String currentItem = dialog.getStringValue(FIELD_GITHUB_CONFIG);
+            Set<String> collaborators = new HashSet<String>();
+            List<BugTrackerGithubConfigParams> configs = getOptions().getConfigs();
+            for(BugTrackerGithubConfigParams config: configs) {
+                if(config.getName().equals(currentItem)) {
+                    collaborators = getCollaborators(config.getName(), config.getPassword(), config.getRepoUrl().split("https://github.com/")[1]);
+                    if(collaborators.size() > 0) {
```

Comment:
getCollaborators can return null, meaning you'll get an NPE. You should either check for null or change it to return an empty Set. I did report this via email...